docs/quickstart/README.md

```diff
@@ -0,0 +1,198 @@
+# SRF Quick Start Guide (QSG)
+
+The SRF Quick Start Guide (QSG) provides examples on how to start using SRF via the Python bindings, C++ bindings or both.
+
+## Prerequisites
+
+- Pascal architecture or better
+- NVIDIA driver `450.80.02` or higher
+- [conda or miniconda](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+- if using docker:
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+
+## Getting Started
+
+To get started with the QSG, it is necessary to get the required SRF components before running any of the examples. The QSG supports two methods for getting the SRF components:
+
+1. Installing via Conda
+   1. Installing via Conda is the easiest method for getting the SRF components and supports both the Python and C++ bindings
+   2. To install SRF via Conda run:
+      ```bash
+      # Change directory to the quickstart root
+      cd ${SRF_HOME}/docs/quickstart/
+
+      # If needed, create a new conda environment
+      conda env create -n srf-quickstart -f environment_cpp.yml
+      conda activate srf-quickstart
+
+      # Or if reusing a conda environment, ensure all dependencies are installed
+      mamba env update -n srf-quickstart -f environment_cpp.yml
+      conda activate srf-quickstart
+
+      # Compile the QSG. This will build the C++ and Hybrid components in ./build. And install the Python and Hybrid libraries
+      ./compile.sh
+      ```
+2. Building from Source
+   1. Installing via the source is for more advanced users and is necessary to try SRF features before an official release.
```

Comment:
maybe link to the CONTRIBUTING.md which is more likely to stay up-to-date

Reply:
Agreed.